Source/WebDriver/WebDriverService.cpp

```diff
@@ -2311,20 +2314,26 @@ static bool processPointerMoveAction(JSON::Object& actionItem, Action& action, s
 
     if (auto xValue = actionItem.getValue("x"_s)) {
         auto x = valueAsNumberInRange(*xValue, INT_MIN);
-        if (!x) {
+        if (!x || (allowFractionalCoordinate == AllowFractionalCoordinate::No && x.value() != std::floor(x.value()))) {
             errorMessage = String("The paramater 'x' is invalid for action"_s);
             return false;
         }
         action.x = x.value();
+    } else {
+        errorMessage = String("The paramater 'x' is missing for action"_s);
```

Comment:
```suggestion
        errorMessage = String("The paramater 'x' is missing for the action."_s);
```
(I see this typo in a few places)